doc/FAQ.rst

```diff
@@ -42,7 +45,7 @@ Yes.
 Can Ginga work with Gtk3?
 -------------------------
 Yes, although the performance is not on par with Gtk2 yet. Cairo for
```

Comment:
This part is out of date.  I will make a PR on your PR.  Gtk3 is the default for gtk backend now and performance is good.

Reply:
On second thought, @pllim, let's merge and I will make a separate small update PR to address these last changes.